cradle-core/src/main/java/com/exactpro/cradle/BookInfo.java

```diff
@@ -472,32 +462,32 @@ public Stream<PageInfo> stream(Instant leftBoundTimestamp, Instant rightBoundTim
     }
 
     private static class PageInterval implements IPageInterval {
+        private final BookId bookId;
         private final BookInfoMetrics.CacheName cacheName;
         private final Map<PageId, PageInfo> pageById;
         private final NavigableMap<Instant, PageInfo> pageByInstant;
 
-        private PageInterval(BookInfoMetrics.CacheName cacheName, Map<PageId, PageInfo> pageById, NavigableMap<Instant, PageInfo> pageByInstant) {
+        private PageInterval(BookId bookId, BookInfoMetrics.CacheName cacheName, Map<PageId, PageInfo> pageById, NavigableMap<Instant, PageInfo> pageByInstant) {
+            this.bookId = bookId;
             this.cacheName = cacheName;
             this.pageById = pageById;
             this.pageByInstant = pageByInstant;
         }
 
-        public BookInfoMetrics.CacheName getCacheName() {
-            return cacheName;
-        }
-
         @Override
         public int size() {
             return pageById.size();
         }
 
         @Override
         public PageInfo get(PageId pageId) {
+            METRICS.incRequest(bookId.getName(), cacheName, GET);
```

Comment:
I think we can pass the whole book ID to the method and get the name inside the method. It will be more informative than having some `String` parameter in the method